### PR TITLE
Minor doc fix for syntax and typo

### PR DESCRIPTION
**Pull request by mauilion**
Signed-off-by: Duffie Cooley <cooleyd@vmware.com>

### CODE REVIEW

**Comment by mauilion** [comment]
my changes are here: https://deploy-preview-16666--kubernetes-io-master-staging.netlify.com/docs/setup/production-environment/tools/kubeadm/high-availability/

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 1]
```diff
@@ -114,21 +114,21 @@ option. Your cluster requirements may need a different configuration.
...
+    Some CNI network plugins like Calico require a CIDR such as `192.168.0.0/16` and
+    some like Weave do not. See the [CNI network documentation](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network).
+    To add a pod CIDR pass the flag `--pod-network-cidr`, or if you are using a kubeadm configuration file
+    set the `podSubnet` field under the `networking` object of `ClusterConfiguration`.
+    {{< /note >}}
 
-    After the command completes you should see something like so:
+    - After the command completes you should see the following:
 
     ```sh
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md]: How about changing these 3 lines to:
```markdown
    The output looks similar to:

    ```
```

**Comment by neolit123** [comment]
LGTM

**Pull request merged**